adopt-a-dog/dogs.html

```diff
@@ -0,0 +1,150 @@
+<!DOCTYPE html>
+<html lang="zxx">
+<head>
+  <meta charset="utf-8">
+  <title>Adopt a Dog</title>
+  <link rel="stylesheet" type="text/css" href="styles.css">
+  
+</head>
+<body>
+    <header>
+            <div class="title">
+                    <!-- <img src = "../adopt-a-dog/images/christopher-harris-57366-unsplash.jpg" alt="brown dog  face in the shadows"  alt = "Christoper harris"class = "header-logo">  -->
+                <img class = "logo" src="adoptadog-logo.svg" alt="ADOPTADOG" width = "400">
```

Comment:
You'll want to move this width out of the HTML and in the css file (also, make it not a hard coded size)